Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -297,6 +306,13 @@ AppDelegateViewModelOutputs {
       )
       .skipNil()
 
+    self.findRedirectUrl = openUrl
```

Comment:
If I'm following correctly, this funnels back into `foundRedirectUrl`, which gets merged into `deepLinkFromUrl` above, so it's one step removed and I think the intention.